### ISSUE TITLE
Generator Creates an Unusual Folder Structure when Running with CLI Arguments

### ISSUE THREAD

**Issue by shurick81** [SharePoint/sp-dev-docs#7466]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [ ] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- browser version
- SPFx version
- Node.js version
- etc

### Describe the bug / error

When using command line arguments, the generator creates this unusual folder structure:

* solution folder
  * config
  * <solution_name>
  * src
  * teams
  * .yo-rc.json

This only happens for 1.13.0 SharePoint generator.

### Steps to reproduce

1. In Windows run these two command:

```
md spfx113testwp00; docker run -it --rm -v ${pwd}/spfx113testwp00:/usr/app/spfx waldekm/spfx:1.13.0 yo @microsoft/sharepoint --solution-name helloworld --framework react --component-type webpart --component-name hello-world-webpart --component-description "HelloWorld web part" --is-domain-isolated true --skip-install --environment spo --skip-feature-deployment false
```

### Expected behavior

When I test with 1.12.1 it creates the following structure:

* solution folder
  * <solution_name>
  * .yo-rc.json

**Comment by patmill** [comment]
This should be addressed in the 1.13.1 drop (coming soon)

**Comment by patmill** [comment]
1.13.1 is now released
